app/api/utility.py

```diff
@@ -47,7 +44,6 @@
 }
 
 CONTEXT = {
-    "cogatlas": "https://www.cognitiveatlas.org/task/id/",
     "nb": "http://neurobagel.org/vocab/",
     "nbg": "http://neurobagel.org/graph/",  # TODO: Check if we still need this namespace.
```

Comment:
Since we're updating the context @neurobagel/dev should we just remove "nbg"? Don't think we use it anywhere, the CLI-generated contexts do not include this either.

Reply:
agree